client/settings/settings-manager/index.js

```diff
@@ -50,9 +49,13 @@ const ExpressCheckoutDescription = () => (
 				'woocommerce-payments'
 			) }
 		</p>
-		<ExternalLink href="https://woocommerce.com/document/payments/settings-guide/#section-4">
-			{ __( 'Learn more', 'woocommerce-payments' ) }
-		</ExternalLink>
+		<a
+			target="_blank"
+			rel="noreferrer"
+			href="https://woocommerce.com/document/payments/settings-guide/#section-4"
+		>
+			Learn more
```

Comment:
If a link opens in a new tab, it _needs_ to have the "(opens in a new tab)" text for accessibility.

Reply:
The text was never meant to be visible to non-SR users - fixing here: https://github.com/Automattic/woocommerce-payments/pull/5568

Reply:
Thanks for pointing that out @frosso! and for the PR to hide the text. I'll revert https://github.com/Automattic/woocommerce-payments/pull/5563/commits/d57076166ea5e483bf1605d1edaafaa9c05d4a86